private/pkg/app/appproto/appprotoexec/appprotoexec.go

```diff
@@ -148,7 +148,11 @@ func NewHandler(
 		}
 		return newProtocProxyHandler(logger, storageosProvider, runner, protocPath, pluginName), nil
 	}
-	return nil, fmt.Errorf("could not find protoc plugin for name %s", pluginName)
+	return nil, fmt.Errorf(
+		"could not find protoc plugin for name %s; Please make sure protoc-gen-%s is installed and present in PATH",
```

Comment:
`could not find protoc plugin for name %s - please make sure protoc-gen-%s is installed and present on your $PATH`